app/controllers/orders_controller.rb

```diff
@@ -100,7 +100,7 @@ def create
         session.delete(:current_delivery_day)
         @grouped_items = @order.items.for_checkout
       else
-        Rollbar.info('Order could not be completed', context: @placed_order.context)
+        Rollbar.warning('Order could not be completed', context_error: @placed_order.context[:error])
```

Comment:
Is `:error` ever a hash or anything besides nil or a string? For my own understanding, are there docs or a way to find an answer to my question in the future?